### ISSUE TITLE
550 HELO is an IP address while trying to send emails from resetting password request but swiftmailer:email:send works ok

### ISSUE THREAD

**Issue by appleandpear** [FriendsOfSymfony/FOSUserBundle#2796]
**Symfony FOSUserBundle versions**:
FOSUserBundle v2.1.2
Symfony v3.4.9
SwiftmailerBundle v2.6.7

**Description of the problem including expected versus actual behavior**:
I am trying to configure swiftmailer for sending resetting password emails properly.
When I am doing test from command line using command `php bin/console swiftmailer:email:send` the message is delivered with success.
But when user is trying to reset password by `/resetting/request` page, he can see the confirmation message:

> An email has been sent. It contains a link you must click to reset your password.
> Note: You can only request a new password once within 2 hours.
> 
> If you don't get an email check your spam folder or try again.

but the e-mail won't be delivered because of error in Logs Profiler. 
I was configured that on my localhost dev enviroment and on my server prod enviroment, but situation is exactly the same.

I must use my own SMTP mail server, I have configured Thunderbird for sending emails from that account with success.

**Steps to reproduce**:
 1. Go to /resetting/request form
 2. Fill in username or email address and click button 'Reset password'
 3. The email with resetting url didn't come to user mailbox

Using console:
 1. Go to project home directory
 2. Type in command: `php bin/console swiftmailer:email:send`
 3. `From` > the same as in `parameters.yml` config
 4. `To` > the same user email address
 5. `Subject` > Something
 6. `Body` > Something else
 7. `[OK] 1 emails were successfully sent.`
 8. The email has been delivered into user email inbox.

**Logs from Profiler**
```
Matched route "fos_user_resetting_send_mail"
Exception occurred while flushing email queue: Expected response code 250 but got code "550", with message "550 HELO is an IP address (See RFC2821 4.1.3) "
```

**E-Mails from Profiler**
Message headers:
```
Content-Transfer-Encoding: quoted-printable
Content-Type: text/plain; charset=utf-8
MIME-Version: 1.0
Date: Wed, 09 May 2018 12:58:27 +0200
Message-ID: <dab609dce9038c477ebd0c435b3d743a@127.0.0.1>
From: Password Resetting <no_reply@myserveraddress.com>
Subject: Reset Password
To: existing_user@existing_domain.com
```

**Config files**:
```
# /app/config/parameters.yml
parameters:
    database_host: 127.0.0.1
    database_port: 3306
    database_name: Symfony
    database_user: symfonyuser
    database_password: p@ssword
    mailer_transport: smtp
    mailer_host: myserveraddress.com
    mailer_user: no_reply@myserveraddress.com
    mailer_password: n0_reply-p@ss
    secret: adfkgljhieughkjhfg
```

```
# /app/config/config.yml
# FosUserBundle Configuration
fos_user:
    db_driver: orm # other valid values are 'mongodb', 'couchdb' and 'propel'
    firewall_name: main
    user_class: AppBundle\Entity\User
    service:
        mailer: fos_user.mailer.twig_swift
    registration:
        confirmation:
            enabled: true
            from_email:
                address: no_reply@myserveraddress.com
                sender_name: Account Register Confirmation
    resetting:
        email:
            from_email:
                address: no_reply@myserveraddress.com
                sender_name: Password Resetting
    from_email:
        address: no_reply@myserveraddress.com
        sender_name: My App
```

**Describe the feature**:

**Comment by stof** [comment]
Well, this is not a FOSUserBundle issue at all. Swiftmailer is failing to send the email, so this should be asked to the Swiftmailer support.

**Comment by appleandpear** [comment]
I have figured out just now, that this issue occurs only when I am runing application under IP address instead of domain name.
So - on my localhost dev enviroment, when I have changed `http://127.0.0.1:8000/` to `http://localhost:8000` the emails are working good.
My production enviroment didn't have domain yet, and that's why there it is not working.

@stof - the Swiftmailer works good from the command line, so that's why I have asked here.

Anyway I think that this issue report might be closed.
